.gitignore

```diff
@@ -27,4 +27,6 @@ UserInterfaceState.xcuserstate
 linters/sass-lint.html
 src/pages/app-blog-post/prerender-blog-data.ts
 src/butter-api/butter-api-key.js
-src/components.d.ts
```

Comment:
This was previously removed - do we need to add this back in?

Reply:
Maybe my gitignore is an old version. I will remove it